archivers/lzlib/files/patch-darwin-Makefile.in.diff

```diff
@@ -0,0 +1,65 @@
+--- ./Makefile.in	2024-03-19 14:11:07
++++ ./Makefile.in	2024-03-19 14:11:43
+@@ -25,14 +25,14 @@
+ lib$(libname).a : lzlib.o
+ 	$(AR) $(ARFLAGS) $@ $<
+ 
+-lib$(libname).so.$(pkgversion) : lzlib_sh.o
+-	$(CC) $(CFLAGS) $(LDFLAGS) -fpic -fPIC -shared -Wl,--soname=lib$(libname).so.$(soversion) -o $@ $<
++lib$(libname).$(pkgversion).dylib : lzlib_sh.o
++	$(CC) $(CFLAGS) $(LDFLAGS) -fpic -fPIC -shared -o $@ $<
```

Comment:
On macOS `-shared` should be changed to `-dynamiclib`.

Reply:
Thank you, changed